easybuild/easyblocks/generic/configuremake.py

```diff
@@ -351,7 +351,8 @@ def build_step(self, verbose=False, path=None):
             ])
             self.log.info("Building target '%s'", target)
 
-            (out, _) = run_cmd(cmd, path=path, log_all=True, simple=False, log_output=verbose)
+            res = run_shell_cmd(cmd, work_dir=path, output_file=verbose)
```

Comment:
`run_shell_cmd` has a different default to always log output in temporary files, and we want to keep doing that also for `ConfigureMake.build_step`

So we should:

* deprecate the `verbose` option for `ConfigureMake.build_step`
* not pass down `verbose` to `run_shell_cmd` at all

Reply:
Done!